### ISSUE TITLE
Can I login to Jina and save all my Doc IDs to Jina?

### ISSUE THREAD

**Issue by kuyin** [jina-ai/discoart#191]
I wonder how to link discoart to my Jina account.

**Comment by gschian0** [comment]
I've been having issues getting the images to persist... any help or any ideas?

**Comment by fa9r** [comment]
@gschian0 Yep, same, I know this used to work sometime last year. I guess the discoart results are no longer automatically stored in docarrays :/
